Dockerfile.viewer

```diff
@@ -0,0 +1,71 @@
+FROM resin/rpi-raspbian:stretch
+MAINTAINER Anton Molodykh <amolodykh@screenly.io>
+
+RUN apt-get update && \
+    apt-get -y install \
+        build-essential \
+        curl \
+        git-core \
+        gstreamer0.10-plugins-good \
+        libdbus-glib-1-dev \
+        libffi-dev \
+        libqt5webkit5 \
+        libraspberrypi0 \
+        libssl-dev \
+        libts-dev \
+        net-tools \
+        omxplayer \
+        psmisc \
+        python-dev \
+        python-netifaces \
+        python-pip \
+        python-setuptools \
+        sqlite3 && \
+    apt-get clean
+
+RUN pip install --upgrade pip
+
+# Install Python requirements
+ADD requirements.txt /tmp/requirements.txt
+ADD requirements.viewer.txt /tmp/requirements.viewer.txt
+RUN pip install -r /tmp/requirements.txt
+RUN pip install -r /tmp/requirements.viewer.txt
+
+# QtBase from packages does not support eglfs
+RUN curl -L https://github.com/antonmolodykh/Screenly-WebVew/releases/download/0.1/qtbase.tar.gz > /tmp/qtbase.tar.gz
```

Comment:
We need to change this to the Screenly hosted one.

Reply:
yes, this is what I mentioned several times